### PR TITLE
updated scripts to handle shutdowns and restarts

### PR DESCRIPTION
**Pull request by cloudnautique**

### CODE REVIEW

File: etcd/0.8.0/rancher-compose.yml [hunk 1]
```diff
@@ -25,10 +25,15 @@ etcd:
...
-    request_line: '/health'
-    interval: 7500
-    response_timeout: 5000
+    port: 1620
+    request_line: '/ping'
+    interval: 10000
+    recreate_on_quorum_strategy_config:
+        quorum: 2
+    strategy: recreateOnQuorum
+    initializing_timeout: 60000
```
Comment (LLParse) [inline etcd/0.8.0/rancher-compose.yml]: edit: I take that back. Let's never timeout on initializing phase; the only way we get stuck in initializing is if etcd has failed, in which case we want to preserve logs and not flap indefinitely

Comment (cloudnautique) [inline etcd/0.8.0/rancher-compose.yml]: Done, thanks for catching.

File: etcd/containers/0.8.0/etcd/run.sh [hunk 2]
```diff
@@ -1,58 +1,45 @@
...
             break
         fi
         sleep 1
     done
-    if [ "$target" == "0" ]; then
-        echo No etcd nodes available
-    else
-        etcdctl --endpoints http://${STACK_NAME}_etcd_$target:2379 $@
+
+    # throw a hail mary... maybe...
```
Comment (LLParse) [inline etcd/containers/0.8.0/etcd/run.sh:21]: haha!

**Comment by LLParse** [comment]
LGTM

**Pull request merged**